core/camel-management-api/src/main/java/org/apache/camel/api/management/mbean/ManagedBacklogDebuggerMBean.java

```diff
@@ -172,4 +172,7 @@
 
     @ManagedOperation(description = "Updates/adds the exchange property (with a new type) on the suspended breakpoint at the given node id")
     void setExchangePropertyOnBreakpoint(String nodeId, String exchangePropertyName, Object value, String type);
+
+    @ManagedOperation(description = "Returns the message history at the given node id as XML")
+    String getMessageHistory(String nodeId);
```

Comment:
Can we rename this to

messageHistorayOnBreakpointAsXml 

because we made a mistake a long time ago that a JMX operation should not be named getter/setter - there are some tools/jconsole etc that dont work well with this.

Reply:
Yes, makes perfect sense. Done.